### ISSUE TITLE
Upgrading from version 4.1.1 to 5.2.0 broke AutoConstructor on subtypes

### ISSUE THREAD

**Issue by jothompson-prismhr** [k94ll13nn3/AutoConstructor#106]
This style of usage worked in version 4.1.1 but now causes the ACONS02 error on the `SubClass` and fails to build when it used to work.

```cs
public interface IDependency
{
}

[AutoConstructor]
public partial class ParentClass
{
    private IDependency Dependency { get; }
}

[AutoConstructor]
public partial class SubClass : ParentClass
{
}
```

**Comment by k94ll13nn3** [comment]
Hello, I can't reproduce this issue with version 5.2.0, both constructors are generated.
Did you reproduce the issue with the code you provided ?

![image](https://github.com/k94ll13nn3/AutoConstructor/assets/10998921/19be21ae-90e0-4d12-916f-fe2cbac57d80)

**Comment by jothompson-prismhr** [comment]
Just tried with the code I provided. It works if everything is in the same project, but fails if SubClass is in a different project. 
![image](https://github.com/k94ll13nn3/AutoConstructor/assets/143096205/7d0f2fa1-4c84-4f93-8bb7-34f7595b1c04)

**Comment by k94ll13nn3** [comment]
Thanks, I will take a loot at it tonight.

**Comment by k94ll13nn3** [comment]
Thanks for the report, fixed in `5.2.1`.
